src/js/renderers/flash.js

```diff
@@ -237,6 +237,7 @@ const FlashMediaElementRenderer = {
 			}
 			;
 		methods.push('stop');
+		methods.push('canplay');
```

Comment:
What happens if you don't add this? This is not a method, but an event. And it's been passed later in the code, so no need for this